src/main/java/net/neoforged/neoforge/event/EventHooks.java

```diff
@@ -688,10 +689,11 @@ public static boolean onProjectileImpact(Projectile projectile, HitResult ray) {
      * which maps to an empty {@link Optional} in {@link LootDataType#deserialize(ResourceLocation, DynamicOps, Object)}
      */
     @Nullable
-    public static LootTable loadLootTable(ResourceLocation name, LootTable table) {
+    @ApiStatus.Internal
+    public static LootTable loadLootTable(ResourceLocation name, LootTable table, HolderLookup.Provider registries) {
         if (table == LootTable.EMPTY) // Empty table has a null name, and shouldn't be modified anyway.
             return null;
-        LootTableLoadEvent event = new LootTableLoadEvent(name, table);
+        LootTableLoadEvent event = new LootTableLoadEvent(registries, name, table);
```

Comment:
How did it end up that you added the lookup provider to the end of the args for `loadLootTable` but the front of the args for `LoadLootTableEvent`?

Reply:
Why _did_ you put the lookup provide at the end of the hook method but at the start of the event constructor? 🤔

Reply:
What makes you think I willingly put them in different places :P I guess it just happened